telegram-bot/meetup.go

```diff
@@ -77,8 +102,13 @@ func addmeetup(ID int64, msgtext string, client mongo.Client) {
 			log.Fatal(err)
 		}
 		bot.Send(tbot.NewMessage(ID, "Meetup added successfully."))
+		nextmeetup(ID, client)
```

Comment:
Why do we call `nextmeetup()` here?

Reply:
Because everytime after adding a meetup the admin would call `/nextmeetup` and then people would keep clicking the blue text and spam the bot. So I thought of doing so . 

Should I change it? :thinking: